### PR TITLE
Rework the packaging scripts for flight-action-api

### PR DESCRIPTION
**Pull request by WilliamMcCumstie**
This PR does three core things:

1. Updates the start/stop/... service scripts with `update_puma_scripts`,
2. Update the postinst,... package scripts with `update_web_suite_package_scripts`, and
3. Remove the integration with `flight service configure`

---

# 1. Update the service scripts

This change is fairly straightforward and uses the standard start/stop scripts for `puma`.

---

# 2. Update the package scripts

This will restart `action-api` on upgrade and stop it on uninstall. Technically `action-api` isn't part of web-suite, however it does integrate with `flight-www`. As such it needs the same package scripts.

---

# 3. Remove the integration with flight service configure

The `postinst` use to integrate with an unreleased `flight-service-1.4.0`. This complexity isn't required and needed to be reworked. Previously `flight service configure` was only used to set the `jwt_secret`.

The `jwt_secret` is set on install and will never need changing in practice. This makes the `flight service configure` integration superfluous. It has been removed in favour of editing `/o/f/o/action-api/config/application.yaml` if it ever needs to be changed.

### CODE REVIEW

File: builders/flight-action-api/config/software/flight-action-api.rb [hunk 1]
```diff
@@ -49,6 +49,12 @@
     copy file, File.expand_path("#{install_dir}/#{file}/..")
   end
 
+  # Patch the development port to the production address
+  block do
+    path = File.join(install_dir, 'config/puma.rb')
+    File.write(path, File.read(path).sub(/^port.*$/, 'bind "tcp://127.0.0.1:917"'))
```
Comment (benarmston) [inline builders/flight-action-api/config/software/flight-action-api.rb:55]: Why is this now needed?  What were we doing to configure this before?

Comment (WilliamMcCumstie) [reply builders/flight-action-api/config/software/flight-action-api.rb:55]: `flight-action-api` has not been ported to use `6304`. The code base does not provide any mechanism to configure this.

https://github.com/openflighthpc/flight-action-api/blob/0fde832a414d5e1a8f680deb3143ecd6f5d7b572/config/puma.rb#L30

Instead, the `bin/start` script use to provide a `--bind` flag to the pumactl command:
https://github.com/openflighthpc/openflight-omnibus-builder/blob/c03e535948228e35596118d7bc088c3fae7d2e87/builders/flight-action-api/opt/flight/opt/action-api/bin/start#L35

---

The newer `bin/start` scripts remove the `--bind` flag as it makes configuring the application easier. Instead the production `bind` address  is directly patched into `config/puma.rb`

Comment (benarmston) [reply builders/flight-action-api/config/software/flight-action-api.rb:55]: I've found it now.  It was previously hard-coded in the start service script.

**Comment by benarmston** [comment]
LGTM.

**Pull request merged**